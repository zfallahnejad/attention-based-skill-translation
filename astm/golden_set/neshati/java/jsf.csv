157882,jsf
203907,jsf
304,jsf
260990,jsf
617373,jsf
1065197,jsf
620338,jsf
650425,jsf
1199132,jsf
472792,jsf
801434,jsf
1530938,jsf
1820286,jsf
100402,jsf
26457,jsf
70604,jsf
139595,jsf
637853,jsf
759126,jsf
870122,jsf
