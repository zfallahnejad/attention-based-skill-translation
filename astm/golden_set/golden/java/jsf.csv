157882,1166,396,0.7464788732394366
203907,104,101,0.5073170731707317
304,78,64,0.5492957746478874
260990,75,80,0.4838709677419355
617373,56,24,0.7
1065197,53,52,0.5047619047619047
620338,50,33,0.6024096385542169
650425,34,34,0.5
1199132,27,17,0.6136363636363636
472792,27,26,0.5094339622641509
801434,19,10,0.6551724137931034
1530938,19,23,0.4523809523809524
1820286,17,23,0.425
100402,15,10,0.6
26457,15,22,0.40540540540540543
70604,14,11,0.56
637853,11,15,0.4230769230769231
759126,11,13,0.4583333333333333
139595,11,3,0.7857142857142857
870122,11,4,0.7333333333333333
354831,10,8,0.5555555555555556
222403,10,5,0.6666666666666666
395181,10,7,0.5882352941176471
